source/strings.tex

```diff
@@ -754,7 +754,7 @@
 @\range{str.data()}{\brk{}str.data() + str.size()}@
 \end{codeblock}
 invalidates pointers, iterators, and references
-returned from \tcode{str}'s member functions.
+to elements of \tcode{str}.
```

Comment:
This has not been applied:
> [Drafting note: The proposed wording also removes the extra code block that previously defined the range]

Reply:
@JohelEGP Which codeblock is that?

Reply:
The one immediately before this line.
![1700185490](https://github.com/cplusplus/draft/assets/21071787/7880ef25-a9a9-4068-8130-8863d61bb6a3)

Reply:
oh, I see. Thanks!